db/db_impl.cc

```diff
@@ -2572,15 +2569,6 @@ Status DBImpl::SetDBOptions(
 #endif  // ROCKSDB_LITE
 }
 
-Status DBImpl::PersistOptions() {
```

Comment:
In DBSetOptions I would like to call HandleWALFull() between EnterUnbatch/ExitUnbatch, making this method useless.